docs/source/developer/interface/command-line-parameters.md

```diff
@@ -0,0 +1,104 @@
+# Command Line Parameters
+
+You can launch Interface with the following parameters.
+
+Displays help dialog
+
+    -?, -h, --help
+
+Displays version information
+
+    -v, --version
+
+Startup at specified URL location
+
+    --url=<value>
+
+Displays the protocol version
+
+    --protocolVersion
+
+Do not show the auto-updater
+
+    --no-updater
+
+Check to see if the machine meets the minimum specification requirements.
+
+    --checkMinSpec
+
+Whether to run the server
+
+    --runserver
+
+(Needs clarification!) Port to listen on
+
+    --listenPort=<port>
+
+Where to find server content
+
+    --serverContentPath=<path>
+
+Set cache location. Defaults to the operating system local data path.
+
+    --cache=<path>
+    
+Set path for defaultScripts.
+
+    --scripts=<path>
+
+Allow multiple instances of the interface to run simultaneously.
+
+    --allowMultipleInstances
+
+Preferred displays.
+
+    --displays=<device>
+
+Disable display plugin. Possible devices include "OpenVR (Vive)", "Oculus Rift"
```

Comment:
```suggestion
Disable display plugin. Possible devices include "OpenVR (Vive)", "Oculus Rift".
```

Additionally, it would be quite nice to terminate all other descriptions with "."s.